src/main/scala/ch/epfl/bluebrain/nexus/kg/async/RevisionValue.scala

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.bluebrain.nexus.kg.async
+
+import akka.cluster.ddata.LWWRegister.Clock
+
+/**
+  * A value with an attached revision corresponding to the value.
+  *
+  * @param rev   the value revision
+  * @param value the value
+  */
+final case class RevisionValue[A](rev: Long, value: A)
```

Comment:
I'd recommend keeping the previous name (_RevisionedValue_) as is better conveys the data type (a value with an attached revision). It's also consistent with _RevisionedId_.